eZ/Publish/Core/FieldType/RichText/Resources/stylesheets/xhtml5/edit/docbook.xsl

```diff
@@ -622,4 +622,27 @@
     <xsl:value-of select="translate( substring-before( substring-after( concat( substring-after( $style, $property ), ';' ), ':' ), ';' ), ' ', '' )"/>
   </xsl:template>
 
+  <xsl:template match="ezxhtml5:div[@data-style]">
```

Comment:
Hmm, this is about our custom styles, not `<div style` attribute, right? If so, then I'd prefer to see here `ezstyle`.